src/main/java/org/jenkinsci/plugins/workflow/job/properties/PipelineTriggersJobProperty.java

```diff
@@ -98,7 +98,24 @@ public void stopTriggers() {
         }
     }
 
+    /**
+     * Starts all triggers.
+     * This method requires the property instance to be added to the {@link WorkflowJob} instance.
+     * Otherwise the triggers will be just skipped.
+     * 
+     * @param newInstance {@code true} if it is a newly created instance.
+     */
     public void startTriggers(boolean newInstance) {
+        if (owner == null) {
+            if (LOGGER.isLoggable(Level.FINE)) {
+                // This is a normal behavior. Even if we miss the triggers,
+                // They will be started once WorkflowJob#addProperty() is called (standard method override).
+                LOGGER.log(Level.FINE, "Cannot start triggers. Owner of the JobProperty has not been assigned yet", 
+                        new IllegalStateException("JobProperty owner is null"));
```

Comment:
@oleg-nenashev 
My thinking here is that generally if a method does not succeed it should throw an exception, not quietly do nothing.  At the same time, null is a possible value and NPEs are bad.  

How about throwing  `IllegalStateException` instead of just logging?

Reply:
My very dim recollection of the situation was that we occasionally get null values due to mistakes elsewhere, or maybe race conditions, so it would be best to log a warning and then move on.